docs/logging.md

```diff
@@ -1,58 +1,68 @@
 # Logging
 
-Using Bigflow, you can easily add a handler to your root logger that will send all logging calls to the Cloud Logging.
-In the CL, you can filter logs per project id/logger name and workflow id. 
 
-## Getting started
-To use logging, you have to install `bigflow` with `logger` module - `bigflow[log]`.
-To create a logger with the CL handler, you have to use `configure_logging` method.
+With Bigflow you can easily configure Python logging to send all messages the Cloud Logging (Stackdriver).
+Cloud Logging allows you to filter logs by project id / log name / workflow id and other fields.
 
-```python
-from bigflow.log import BigflowLogging
-project_id = 'some_project'
-logger_name = __name__
-
-
-BigflowLogging.configure_logging(project_id, logger_name)
-```
+To use logging, you have to install `bigflow` with `log` module - `bigflow[log]`.
+Then you need to pass logging configuration to your `Workflow`.
 
-or, if you prefer to filter CL logs by workflow id:
- 
 ```python
-from bigflow.log import BigflowLogging
-project_id = 'some_project'
-logger_name = __name__
-workflow_id = 'some-workflow'
+import bigflow
+
+gcp_project_id = 'some_project_id'
+
+workflow = bigflow.Workflow(
+    name='workflow_name',
+    log_config={
+        'project_id': gcp_project_id,
+        'level': 'DEBUG',
+    },
+    # ...
+)
+```
 
+BigFlow will automatically configure `logging` when workflow is executed with `bigflow run` or `bigflow deploy` commands.
```

Comment:
I don't like `automatically` idea. Maybe not every user want logs in CL.

Reply:
Logging is not configured out of the box (unfortunately, we don't have the project_id to do that). 

Btw. default, reasonable configuration is a good idea in my opinion.